apps/frontend/src/components/xpress/XpressProposalTable.tsx

```diff
@@ -0,0 +1,363 @@
+import { useSearchParams } from 'react-router-dom';
+
+import { ProposalsFilter } from 'generated/sdk';
+import { useCallsData } from 'hooks/call/useCallsData';
+import { useProposalStatusesData } from 'hooks/settings/useProposalStatusesData';
+import { useTechniquesData } from 'hooks/technique/useTechniquesData';
+import { StyledContainer, StyledPaper } from 'styles/StyledComponents';
+import {
+  addColumns,
+  fromArrayToCommaSeparated,
+  setSortDirectionOnSortField,
+} from 'utils/helperFunctions';
+import { tableIcons } from 'utils/materialIcons';
+
+import { ProposalViewData, useProposalsCoreData } from './useProposalsCoreData';
+import { useXpressInstrumentsData } from './useXpressInstrumentsData';
+import XpressProposalFilterBar from './XpressProposalFilterBar';
+
+const XpressProposalTable = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [selectedProposals, setSelectedProposals] = useState<
+    ProposalViewData[]
+  >([]);
+  const { techniques, loadingTechniques } = useTechniquesData();
+  const { proposalStatuses, loadingProposalStatuses } =
+    useProposalStatusesData();
+  const [currentPage, setCurrentPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(20);
+  const PREFETCH_SIZE = 200;
+
+  const [searchParams, setSearchParams] = useSearchParams({});
+
+  const callId = searchParams.get('callId');
+  const instrument = searchParams.get('instrument');
+  const technique = searchParams.get('technique');
+  const proposalId = searchParams.get('proposalId');
+  const to = searchParams.get('to');
+  const from = searchParams.get('from');
+  const proposalStatusId = searchParams.get('proposalStatusId');
+  const search = searchParams.get('search');
+  const selection = searchParams.getAll('selection');
+  const sortField = searchParams.get('sortField');
+  const sortDirection = searchParams.get('sortDirection');
+
+  type QueryParameters = {
+    query: {
+      first?: number;
+      offset?: number;
+    };
+    searchText?: string | undefined;
+  };
+  const [queryParameters, setQueryParameters] = useState<QueryParameters>({
+    query: {
+      first: PREFETCH_SIZE,
+      offset: 0,
+    },
+    searchText: search ?? undefined,
+  });
+
+  const [proposalFilter, setProposalFilter] = useState<ProposalsFilter>({
+    callId: callId ? +callId : undefined,
+    instrumentFilter: {
+      instrumentId: instrument ? +instrument : null,
+      showAllProposals: !instrument,
+      showMultiInstrumentProposals: false,
+    },
+    techniqueFilter: {
+      techniqueId: technique ? +technique : null,
+      showAllProposals: !technique,
+      showMultiTechniqueProposals: false,
+    },
+    dateFilter: {
+      to: to ? to : null,
+      from: from ? from : null,
+    },
+    referenceNumbers: proposalId ? [proposalId] : undefined,
+    proposalStatusId: proposalStatusId ? +proposalStatusId : undefined,
+    text: search,
+    excludeProposalStatusIds: [9],
```

Comment:
This is for filtering out the expired proposals while displaying the proposals for technique/instrument scientists. Other xpress proposals are expected to have only the xpress specific statuses.

Reply:
For the proposal status we might need to only show those related to Xpress proposals.

Reply:
Yes, we should only show xpress statuses, this will be handled as part of upcoming issue for xpress status implementation (Once the statuses are confirmed).

Reply:
I thinks we have some which are relevant and then we can add the additional as they are decided. I can see we have some which are being included and may not be relevant to these proposals.

Reply:
I'm currently working on this issue to be able to change statuses and exclude the ones which aren't applicable so it will be changed very soon

Reply:
Can we have a comment on this if it will be fixed through another issue and we can also include the issue number please.

Reply:
Status related logics will modified/fixed as part of https://github.com/UserOfficeProject/issue-tracker/issues/1138.